critiquebrainz/frontend/external/musicbrainz_db/serialize.py

```diff
@@ -83,19 +83,56 @@ def to_dict_places(place, includes={}):
     return data
 
 
+def to_dict_artist_credit_names(artist_credit_name):
+    data = {
+        'name': artist_credit_name.name,
+        'artist': to_dict_artists(artist_credit_name.artist),
+    }
+    if artist_credit_name.join_phrase:
+        data['join_phrase'] = artist_credit_name.join_phrase
+    return data
+
+
 def to_dict_release_groups(release_group, includes={}):
```

Comment:
I think that we should have tests for these conversion methods to make sure that we correctly create a dict from a particular releasegroup id.
I think that this will be a bit difficult - we can't really do the query against a real musicbrainz database because the data might change in the database. I'm not sure that it's worth testing only this function, I'd like to see it tested in conjunction with `fetch_multiple_release_groups`
Maybe @gentlecat has a suggestion here?